OmiseSDK/AtomeFormViewController.swift

```diff
@@ -207,11 +207,16 @@ private extension AtomeFormViewController {
             return
         }
 
+        let shippingAddress =
```

Comment:
@vault087 Do you mean to have Atome changes as part of this PR?

Reply:
Unfortunately this change accidentally come to this PR as this tasks was part of Atome. It doesn't affect Atome anyhow though so I just leave it here, it's already fixed in latest related PRs